test/e2e/flowaggregator_test.go

```diff
@@ -852,7 +853,9 @@ func getUint64FieldFromRecord(t *testing.T, record string, field string) uint64
 func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, isDstService bool, checkAllRecords bool, isIPv6 bool) (string, []string) {
 	var collectorOutput string
 	var recordSlices []string
-	err := wait.PollImmediate(500*time.Millisecond, aggregatorInactiveFlowRecordTimeout, func() (bool, error) {
+	// In the ToExternalFlows test, flow record will arrive 5.5s (exporterActiveFlowExportTimeout+aggregatorActiveFlowRecordTimeout) after executing wget command
+	// We set the timeout to 8s (exporterActiveFlowExportTimeout+aggregatorInactiveFlowRecordTimeout) to make the ToExternalFlows test more stable
```

Comment:
It seems reasonable to me to wait for 8s, which is slightly larger than the expected delay (5.5s). However, do we choose this value _because_ it is slightly larger than 5.5s or doe choose it because it is `exporterActiveFlowExportTimeout+aggregatorInactiveFlowRecordTimeout`? In other words, if `aggregatorInactiveFlowRecordTimeout` was larger (e.g. 20s), would be still be using `exporterActiveFlowExportTimeout+aggregatorInactiveFlowRecordTimeout`?

Reply:
That sounds good, thanks

Reply:
Cool, updated.